### ISSUE TITLE
Audio don't work in High Sierra 12.13

### ISSUE THREAD

**Issue by cleristonmartins** [toleda/audio_CloverALC#121]
I installed Sierra but my audio does not work, I already installed Native AppleHDA and tried to run the toleda patch only that it does not work.

Can someone help me?

Corporation 8 Series / C220 Series High Definition Audio Controller Chipset

**Comment by toleda** [comment]
Attach cloverALC script output

**Comment by cleristonmartins** [comment]
Realtek ALC668 is not supported with audio_cloverALC-130.command_v0.3

![screen shot 2017-10-09 at 7 18 22 pm](https://user-images.githubusercontent.com/17992006/31366583-ad44aa56-ad26-11e7-8734-a904a5b87a29.png)

**Comment by toleda** [comment]
See README/.../Supported codecs
ALC668 is not supported.

**Comment by cleristonmartins** [comment]
I installed the ALC888 codec but I am having the error as the print, I am confused about the native codec .. My native is ALC 668 in which the script does not support, what do you do?

![screen shot 2017-10-09 at 9 15 19 pm](https://user-images.githubusercontent.com/17992006/31369117-0c8365ec-ad37-11e7-9496-7c52c812843d.png)

**Comment by toleda** [comment]
script is reporting AppleHDA.kext is not native, restore native AppleHDA.kext
ALC888 solution will never work for ALC668.
Script has no support for ALC668.

**Comment by cleristonmartins** [comment]
So you're telling me that I should get the native ALC888? Because my native AppleHDA.kext is ALC668, which the script does not support.

**Comment by toleda** [comment]
No. You need ALC668.  The script does not support ALC668. cloverALC is not a solution for your system.

**Comment by cleristonmartins** [comment]
So you have some solution for this?

**Comment by toleda** [comment]
Nothing specific to 668. Try AppleALC.kext or Voodoo.

**Comment by cleristonmartins** [comment]
resolved, that extract the dsdt from the machine with linux and audi running, then put in the folder patche in the clover and it worked.
